src/analysis/convex/topology.lean

```diff
@@ -69,9 +69,36 @@ end std_simplex
 
 /-! ### Topological vector space -/
 
+section linear_ordered_ring
+variables [linear_ordered_ring 𝕜] [densely_ordered 𝕜] [topological_space 𝕜] [order_topology 𝕜]
+  [linear_ordered_add_comm_group E] [topological_space E] [order_topology E] [module 𝕜 E]
```

Comment:
`order_topology E` feels like rather a strong assumption. This pretty much only applies in the 1D case, right?

Reply:
Indeed, I got fooled by a `continuity` failure. `[t2_space E] [has_continuous_add E]` is enough.